ledger/participant-integration-api/src/main/resources/db/migration/oracle-appendonly/V1__Append_only_schema.sha256

```diff
@@ -1 +1 @@
-fe940b22140c0a1da0a3e54c0087c8872828e4cb08bdcc30bbaa1db84f98b62b
+93dd1c0237cb7f8b193aef03c88d3fd22e85c5e8d2c26adeff4c6eeeba0d500c
```

Comment:
Oracle already has [10 migrations](https://github.com/digital-asset/daml/tree/main/ledger/participant-integration-api/src/main/resources/db/migration/oracle-appendonly), why are you changing the first one? Shouldn't you add a new migration, just like Postgres?

Reply:
Oh, how did I miss that.

Reply:
Thanks for catching this. Amended